lib/train/k8s/container/connection.rb

```diff
@@ -8,12 +8,16 @@ class Connection < Train::Plugins::Transport::BaseConnection
 
         # URI format: k8s-container://<namespace>/<pod>/<container_name>
         # @example k8s-container://default/shell-demo/nginx
+
+        DEFAULT_NAMESPACE = "default"
```

Comment:
actually this is not needed.. we already have `Train::K8s::Container::KubectlExecClient::DEFAULT_NAMESPACE`